website/config.toml

```diff
@@ -0,0 +1,19 @@
+# The URL the site will be built for
+# TODO(@cpu): set up https://ffi.rustls.dev DNS
+base_url = "https://rustls.github.io/rustls-ffi/"
```

Comment:
FWIW I've set up the DNS for this

Reply:
Thank you! It seems to be showing as "DNS Check in Progress" in the [repo settings page](https://github.com/rustls/rustls-ffi/settings/pages). I'll check again in a bit.

Reply:
✅  DNS check successful

Reply:
Updated the site config to use `ffi.rustls.dev`.